README.md

```diff
@@ -19,6 +20,7 @@
 
 ## Current News
 * November 14, 2023 - [Announcing .NET MAUI in .NET 8](https://devblogs.microsoft.com/dotnet/announcing-dotnet-maui-in-dotnet-8/)
+* November 8, 2024 - [Announcing .NET MAUI in .NET 9](https://learn.microsoft.com/en-us/dotnet/maui/whats-new/dotnet-9?view=net-maui-9.0)
```

Comment:
@davidortinau Do we want to just leave the latest big release blog post (.NET 9 now)?